cli_tools/gce_image_publish/publish/publish.go

```diff
@@ -473,22 +491,36 @@ func (p *Publish) deprecatePrintOut(deprecateImages *daisy.DeprecateImages) {
 
 	for _, di := range *deprecateImages {
 		image := path.Base(di.Image)
-		switch di.DeprecationStatus.State {
+		switch di.DeprecationStatusAlpha.State {
 		case "DEPRECATED":
 			p.toDeprecate = append(p.toDeprecate, image)
 		case "OBSOLETE":
 			p.toObsolete = append(p.toObsolete, image)
-		case "":
+		case "ACTIVE", "":
```

Comment:
Was this a bug earlier or a downstream dependency changed?

Reply:
In a rollback we would previously set it to empty string as ACTIVE and empty string in the deprecation policy are the same value.

Since the daisy module switches to the Alpha API when this field is set, I changed rollbacks to set it to active instead of empty set.